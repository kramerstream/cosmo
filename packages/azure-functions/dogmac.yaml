uses:
  assert:
    js: assert
  doubles:
    js: @cosmokramer/doubles
  expected:
    js: @cosmokramer/expected
  fs:
    js: @dogmalang/fs.sync
  isEqual:
    js: lodash.isequal
  path:
    js: @dogmalang/path
