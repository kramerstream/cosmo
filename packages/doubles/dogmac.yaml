uses:
  assert:
    js: assert
  expected:
    js: @cosmokramer/expected
  fs:
    js: @dogmalang/fs.sync
  isEqual:
    js: lodash.isequal
  path:
    js: @dogmalang/path
