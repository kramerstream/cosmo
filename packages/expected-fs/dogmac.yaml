uses:
  assert:
    js: assert
  expected:
    js: @cosmokramer/expected
  fs:
    js: @dogmalang/fs.sync
  path:
    js: @dogmalang/path
