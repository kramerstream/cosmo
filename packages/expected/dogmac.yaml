uses:
  assert:
    js: assert
  get:
    js: lodash.get
  path:
    js: @dogmalang/path
